### PR TITLE
AMD Default GPU

### PR DESCRIPTION
**Pull request by TechnoShip123**
I've added resources and a markdown file in their respective locations for setting the default GPU on AMD graphics cards. @duendwang can you pull this into the repository? I've also matched it with the directory structure.

### CODE REVIEW

**Comment by TechnoShip123** [comment]
Forgot to delete the `deleteme` files, they were temporary so that I could create directories that matched master's dir format. I'll fix it asap.

File: _help/amd-dedicated-gpu/dedicated_gpu.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# How to Switch Between Graphics Card and iGPU with AMD
```
Comment (LordRalex) [inline _help/amd-dedicated-gpu/dedicated_gpu.md:1]: This is missing the default header that other pages have.

File: _help/amd-dedicated-gpu/dedicated_gpu.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+#### Question:
+I have a good graphics card, but when I play Minecraft I get a low amount of FPS such as 20, when I should be getting a high amount like 100. Is this a problem with my graphics card or some sort of bug?
+
+#### Answer:
+Some computers have a processor that also contains its own GPU, commonly known as the **iGPU** (Integrated Graphics Processing Unit). While it functions like most GPUs, it isn't as strong as most graphics cards. Getting low FPS despite having a good graphics card could be due to the computer using the iGPU instead of your graphics card. If you have an AMD graphics card, follow the below steps to set your dedicated GPU.
```
Comment (LordRalex) [inline _help/amd-dedicated-gpu/dedicated_gpu.md:7]: Nvidia has the same issue. Any reason why it's not included here, and why this page is a generic question, but it's specific to a solution.

File: static/images/help/amd-dedicated-gpu/deleteme.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+e
```
Comment (LordRalex) [inline static/images/help/amd-dedicated-gpu/deleteme.txt:1]: Yes, delete this.

File: _help/amd-dedicated-gpu/dedicated_gpu.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+![](/static/images/help/amd-dedicated-gpu/gpu_workload.png)
+
+> **4)** Click OK to restart the Radeon Software and the change will take effect.
+
+![](/static/images/help/amd-dedicated-gpu/rsx_restart.png)
+
+After that, you should be good to go! Open the Minecraft Launcher, start Minecraft, and you should get much better performance!
+
+**TIP:** For even *better* performance, make sure to consider mods such as [Sodium](http://www.curseforge.com/minecraft/mc-mods/sodium) and [Optifine](http://optifine.net/home)!
```
Comment (LordRalex) [inline _help/amd-dedicated-gpu/dedicated_gpu.md:33]: I am not a fan of recommending 3rd party content here, especially with the issues they can cause. This is an unrelated help that I would prefer we not include on this page.

**Review by LordRalex — CHANGES_REQUESTED** [review]

File: _help/amd-dedicated-gpu/dedicated_gpu.md (hunk 2 above)
Comment (TechnoShip123) [reply _help/amd-dedicated-gpu/dedicated_gpu.md:14]: @LordRalex I'm not exactly sure what you meant by this, does [this commit](https://github.com/TechnoShip123/MinecraftHopper/commit/09c1da21e0485487fbdedfc8e7fa8a221c3e75b3) solve what you said?

**Review by LordRalex — APPROVED** [review]

**Pull request merged**